pkgs/development/python-modules/visions/default.nix

```diff
@@ -0,0 +1,75 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pythonOlder
+, pytestCheckHook
+, attrs
+, imagehash
+, matplotlib
+, multimethod
+, networkx
+, numpy
+, pandas
+, pillow
+, pydot
+, pygraphviz
+, shapely
+}:
+
+buildPythonPackage rec {
+  pname = "visions";
+  # latest release (Dec 2021) requires barely used 1.8GB `tangled-up-in-unicode` packae
+  version = "unstable-2022-10-30";
+  disabled = pythonOlder "3.6";
+
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "dylan-profiler";
+    repo = pname;
+    rev = "a0b55bbf95e6efe001195e4b497358d6283966b5";
+    hash = "sha256-PF2h+05WuNi7lZUF+phvBjjr7yxGkPjnzHKLw0zqkDQ=";
+  };
+
+  propagatedBuildInputs = [
+    attrs
+    imagehash
+    multimethod
+    networkx
+    numpy
+    pandas
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+  ] ++ passthru.optional-dependencies.type-geometry
+    ++ passthru.optional-dependencies.type-image-path
+    ++ passthru.optional-dependencies.plotting;
+
+  # errors due to use of deprecated Numpy API:
+  disabledTests = [ "test_declarative" "test_sequences" ];
+  disabledTestPaths = [
+    # errors due to use of deprecated Numpy API:
```

Comment:
Is this maybe related to changed pytest settings in setup.cfg or similar?

Reply:
No longer applicable.